api/src/main/java/org/openmrs/module/appframework/service/AppFrameworkServiceImpl.java

```diff
@@ -346,9 +343,13 @@ public boolean checkRequireExpression(Requireable candidate, AppContextModel con
 					} else {
 						bindings.put(e.getKey(), e.getValue());
 					}
-				}
+			}
+
+			for (Map.Entry<String, Object> e : contextModel.entrySet()) {
+				String jsonValue = new ObjectMapper().writeValueAsString(e.getValue());
+				javascriptEngine.eval(e.getKey() + " = " + jsonValue);
+			}
 
-				javascriptEngine.setBindings(bindings, ScriptContext.ENGINE_SCOPE);
```

Comment:
What could be the reason behind removing the above line?

Reply:
Keeping this line cause the following error.

```
ERROR - AppFrameworkServiceImpl.checkRequireExpression(382) |2023-11-14 18:07:03,377| Failed to evaluate 'require' check for extension null
java.lang.IllegalArgumentException: The value 'com.oracle.truffle.js.runtime.truffleinterop.InteropBoundFunction@9a265320' cannot be passed from one context to another. The current context is 0x7194dae5 and the argument value originates from context 0x82300e1.
```

Reply:
At runtime or compile time?